filebeat/magefile.go

```diff
@@ -220,3 +224,57 @@ func PythonIntegTest(ctx context.Context) error {
 		return devtools.PythonNoseTest(args)
 	}, "GENERATE", "TESTING_FILEBEAT_MODULES", "TESTING_FILEBEAT_FILESETS")
 }
+
+// CreateModule creates a new Filebeat module.
+// Use MODULE=module to specify the name of the new module
+func CreateModule() error {
+	targetModule := os.Getenv("MODULE")
+	if targetModule == "" {
+		return fmt.Errorf("You must specify the module: MODULE=name mage createModule")
+	}
+
+	filebeatDir, err := os.Getwd()
```

Comment:
Use `devtools.CWD()` as a shortcut. It will panic on error, but that's how errors bubble up in mage.